docs/classifier_example/classification_v4_4_features.ipynb

```diff
@@ -0,0 +1,884 @@
+      "text/plain": [
+       "<Figure size 1152x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAPcAAAF8CAYAAAATjC38AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAF5NJREFUeJzt3X+UXHV5x/H3JoFdownHEjVVj8V4Mg+LQtCNmGCAIgYqlgMW5Si1SGyQH1JFqhahcICqVD2B1h9obRSBglalqRRMpCKKJkaapUIw4SFROfiDiAtqoGEXkkz/uHfI7OzMzszufDczz3xe5+zJzL3PfO+TTT733rnznZmeYrGIiMQzbW83ICJpKNwiQSncIkEp3CJBKdwiQSncIkHN2NsNSFpmVgTuA3YBRWAmsB042903JNre84CngVXu/rpWb0Mao3B3h6Pdfah0x8zeD3wKWJxwm88FDks4vtShcHcZM5sBvAR4rGzZRcDJZE/THgTOcfdfm9lfAH8P7CY78n/A3e80s+8Cn3b3r+ePH3U/dw3wLDP7MTAAXAK8CXgKeBQ43d0fTvhX7Xp6zt0d7jCze83s18AD+bJlAGZ2GnAwcJi7Hwp8E1iZ13yCLOgLgYuBP21im8uAJ/MxXwicB7w6H+s24DWT+ytJPTpyd4ej3X3IzF5FFt473P2RfN2fk50+bzAzgOlkz8sBvgKsMrNbgf8GPj7B7f8KuAe428xWA6vd/fYJjiUN0pG7i7j73cD7gC+Z2QH54unAx9z90PwouxB4bV5/EbAE2ACcDtyZP6YI9JQNvW+d7e4GjsrHeBS4yswmuqOQBincXcbdvwzcBVyVL/oWsNzMZuf3LweuN7MZZvYgMNPdPwecAxxiZr3Ab8l2ApjZQcAhVTa1E5huZj1mtoDsiv1md78i3/arU/z9ZA+dlnenc4F7zew4sufXLwLW5y9jPUR2sWunmZ0H3GhmT5NdVHunu4+Y2YeBa83sjcD97Dmil3uYbCfyE+AI4Ktkp/5PAE8C70n7V5QeveVTJCadlosEpXCLBJXkObeZTQOuBhYAI8Byd99atv6TZFdkH88Xnejuf0jRi0i3SnVB7SSgz90Xm9kiYAVwYtn6VwHHlU+JFJHWSnVavgRYA+Du68lfNoFnjurzgc+b2Voze2eiHkS6Wqoj92yg/DR7l5nNcPedwLPJ3rRwJdkEijvMbIO731s+wODgoC7jizRgYGCgp+qKYrHY8p9CoXBloVA4pez+L8tuTy8UCrPK7n+8UCj8VeUYGzZsKDZr06ZNbVPfTr00W69e2r+XkjwnVXOY6rR8LXA8QP6ce2PZugLwAzObbmb7kJ3C352oD5Guleq0fBWw1MzWkc1BXmZm5wNb3f1mM7sBWE
```

Comment:
Here the error is just a string, above it is always IndexError

Reply:
Also ValueError